packages/venia-ui/lib/components/MiniCart/miniCart.js

```diff
@@ -63,13 +64,27 @@ const MiniCart = React.forwardRef((props, ref) => {
         }
     }, [addToast, errors]);
 
+    const header =
+        loading || !subTotal ? (
+            'Loading...'
```

Comment:
Showing "Loading" in place of the entire header component differs from the direction we went on the cart page where we "blanked" the values while the mutation is in flight. Was this intentional?


**Removing an item in mini cart** 
[![Image from Gyazo](https://i.gyazo.com/dbe2b9b647b761098e248f6230e2ce1f.gif)](https://gyazo.com/dbe2b9b647b761098e248f6230e2ce1f)


**Removing an item on the cart page**
[![Image from Gyazo](https://i.gyazo.com/47fd2ad66ff469ad8c042dce5e1c2162.gif)](https://gyazo.com/47fd2ad66ff469ad8c042dce5e1c2162)

Reply:
This stemmed out of the dev team dicussion we had a couple of weeks ago before the break. @soumya-ashok @schensley is this expected?

Reply:
@sirugh and @revanth0212 - Don't remember ever seeing "Loading..." before and it looks very strange in the header.

Reply:
What do you think should be the placeholder value when an operation is in progress?

Reply:
> What do you think should be the placeholder value when an operation is in progress?

IMO the item count and cart value should remain while the operation is in flight (item is greyed out) until the operation finishes (the item gets removed from the cart). At that point the header should update to reflect the new item count and cart value.

Reply:
@soumya-ashok what do you think?

Reply:
I agree with @sirugh - the only addition I'd make is that it would be nice to have a smooth transition when the totals update - perhaps opacity reduces a bit and comes back just to provide visual feedback that there has been a change.

Reply:
I have made the change to not show the `Loading...` text while updating. Unfortunately, I am not aware of how to make that a smooth transition. Since it is the same component that is updating, I am not aware of a CSS trick to achieve the transition. @jimbo any thoughts?

Reply:
> This stemmed out of the dev team dicussion we had a couple of weeks ago before the break.

> Don't remember ever seeing "Loading..." before

> What do you think should be the placeholder value when an operation is in progress?

We have only two loading patterns:

1. A [blocking indicator](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/LoadingIndicator/static.js), used by replacing content with a spinning icon and `Fetching` text
2. A [non-blocking indicator](https://github.com/magento/pwa-studio/blob/8c187738be36df7572261d21199f213b7e8cb50c/packages/venia-ui/lib/components/PageLoadingIndicator/indicator.js), used by inserting only a rotating icon into a header or something

Not sure where the `Loading...` concept came from here; the goal was to add an icon to this header. As for whether to show the price & total while they are updating, we did discuss hiding them. If UX prefers to gray them out, that is also fine.

Animating such a transition should be fairly simple:

```stylus
.foo
  opacity: 1
  transition: opacity linear

.foo_loading
  composes: foo
  opacity: 0
```

Reply:
Nice. Thanks for the input. Handled in be5563a.